app.rb

```diff
@@ -1,9 +1,30 @@
 require 'sinatra/base'
+require 'sinatra/reloader'
+require 'pg'
+require './lib/peep'
 
 class Chitter < Sinatra::Base
+  configure :development do
+    register Sinatra::Reloader
+  end
+
   get '/test' do
     'Test page'
   end
 
+  get '/' do
```

Comment:
Instead of immediately redirecting to Peeps, you could try loading the peeps here instead. It will still meet the specification this way, as the Peeps will display on the homepage. This way, you would need one less route, which is slightly redundant in this instance.